File: src/com/ptby/dynamicreturntypeplugin/config/multi/RefreshProjectCallBack.java
package com.ptby.dynamicreturntypeplugin.config.multi;

import com.intellij.openapi.project.Project;
import com.ptby.dynamicreturntypeplugin.config.DynamicReturnTypeConfig;

public interface RefreshProjectCallBack {

    public void setConfig( Project project, DynamicReturnTypeConfig config );

}

File: src/com/ptby/dynamicreturntypeplugin/config/multi/RefreshProjectCallBack.kt
package com.ptby.dynamicreturntypeplugin.config.multi

import com.intellij.openapi.project.Project
import com.ptby.dynamicreturntypeplugin.config.DynamicReturnTypeConfig

interface RefreshProjectCallBack {

    fun setConfig(project: Project, config: DynamicReturnTypeConfig)

}
